bin/adb-reconnect.sh

```diff
@@ -2,11 +2,5 @@
 
 while read d
 do
-  if [ -z "$STF_PROVIDER_PUBLIC_IP" ]; then
-    # real devices
-    timeout -t 3 adb -s $d reboot
-    timeout -t 3 adb -s $d reconnect
-  else
-    date > /tmp/reboot
-  fi
+    adb -s $d reconnect
```

Comment:
i would keep the timeout to ensure quicker workaround, because adb command can easily get stuck for a long time

Reply:
Added timeout